js/src/main/scala/com/raquo/domtypes/jsdom/defs/package.scala

```diff
@@ -86,6 +86,10 @@ package object defs {
     type EmbedTags[T[_ <: dom.Element]] = generic.defs.tags.EmbedTags[T, dom.Element, dom.html.Image, dom.html.IFrame, dom.html.Embed, dom.html.Object, dom.html.Param, dom.html.Video, dom.html.Audio, dom.html.Source, dom.html.Track, dom.html.Canvas, dom.html.Map, dom.html.Area]
     type TableTags[T[_ <: dom.Element]] = generic.defs.tags.TableTags[T, dom.Element, dom.html.Table, dom.html.TableCaption, dom.html.TableCol, dom.html.TableSection, dom.html.TableRow, dom.html.TableCell, dom.html.TableHeaderCell]
     type MiscTags[T[_ <: dom.Element]] = generic.defs.tags.MiscTags[T, dom.Element, dom.html.Title, dom.html.Style, dom.html.Element, dom.html.Quote, dom.html.Progress, dom.html.Menu]
+    import org.scalajs.dom._
+    import svg._
```

Comment:
please don't use those extra imports – instead, spell out `dom.svg.Type` everywhere instead of just `Type` – yes, even in this huge list. It's just much easier to text search, and matches the style in the rest of the definitions here.